proposals/aip-1.md

```diff
@@ -0,0 +1,229 @@
+
+If you are interested in assuming ownership of an AIP, send a message asking to take over, addressed to both the original author and the AIP editor. If the original author doesn't respond to email in a timely manner, the AIP editor will make a unilateral decision (it's not like such decisions can't be reversed :).
+
+
+# Proposal Editors #
+The current AIP editors are members of the [@Agorex-io/proposal-editors] team.
+
+
+# Proposal Editor Responsibilities and Workflow #
+For each new AIP that comes in, an editor does the following:
+
+- Read the AIP to check if it is ready: sound and complete. The ideas must technically make sense, even if they don't seem likely to be accepted.
+- The title should accurately describe the content.
+- Edit the AIP for language (spelling, grammar, sentence structure, etc.), markup (Github flavored Markdown).
+
+If the AIP isn't ready, an editor will send it back to the author for revision, with specific instructions.
+
+Once the AIP is ready for the repository, a AIP editor will:
+
+- Assign an AIP number (generally the PR number or, if preferred by the author, the Issue number if there was discussion in [the Issues section of this repository] about this AIP).
+
+- Accept the corresponding pull request (pending approved reviews from the Board as described in _Proposal Workflow_ above).
+
+- Send a message back to the AIP author with the next step (tagging them in an Issue comment or PR is sufficient).
+
+AIP editors don't pass judgment on AIPs. They merely do the administrative & editorial part (which is generally a low volume task).
+
+
+# History #
+This document was derived heavily from [Ethereum's EIP 1] which was derived from [Bitcoin's BIP-0001] written by Amir Taaki which in turn was derived from [Python's PEP-0001]. In many places text was simply copied and modified. Although the EIP-1 text was written by Martin Becze and Hudson Jameson, the PEP-0001 text was written by Barry Warsaw, Jeremy Hylton, and David Goodger, they are not responsible for its use in the Agorex Improvement Process, and should not be bothered with questions specific to Agorex or the AIP. Please direct all comments to the AIP editors.
+
+
+# Assumptions #
+- Assumes 2FA is required by the GitHub org and all directors on the Board of Directors are using 2FA when submitting PRs, commits, and reviewing proposals.
+- Roles:
+  - ONLY the [@Agorex-io/board-of-directors] can approve proposals that move beyond a PR and end up merged to the repository. 
+  - [@Agorex-io/proposal-editors] cannot approve, but can facilitate the process and guide champions and authors. They can also edit PRs in progress to help get the preamble and formatting correct according to the workflow. 
```

Comment:
In the proposal editor section, it states that the editor will "accept the corresponding pull request". This section seems to state the obvious? Should they mark it as complete, notify a board member, and the board member can pull?

Reply:
I reworded the proposal editor section slightly to be more explicit and removed this Assumptions section. The Assumptions section was really more my notes and comments for you, which were later addressed in the core doc so they became repetitive. I don't think appropriate to leave in the doc permanently anyway.